src/main/java/com/example/App.java

```diff
@@ -7,9 +7,9 @@
 
 // org.crac could be used instead of jdk.crac
 // https://github.com/CRaC/docs#orgcrac
-import jdk.crac.Context;
-import jdk.crac.Core;
-import jdk.crac.Resource;
+import org.crac.Context;
```

Comment:
There is a lightweight testing that all combinations are working: https://github.com/CRaC/example-jetty/blob/master/.github/workflows/maven.yml#LL60C47-L60C50

Those scripts then should be fixed as well.

Probably you need to allow GHA to run on your personal fork to verify the matrix succeed.